flowetl/flowetl/flowetl/operators/staging/sql/cleanup_staging_table.sql

```diff
@@ -0,0 +1,12 @@
+BEGIN;
+
+DROP TABLE staging_table_{{ds_nodash}};
+DROP FOREIGN TABLE IF EXISTS call_table_{{ds_nodash}};
+DROP FOREIGN TABLE IF EXISTS sms_table_{{ds_nodash}};
+DROP FOREIGN TABLE IF EXISTS location_table_{{ds_nodash}};
+DROP FOREIGN TABLE IF EXISTS mds_table_{{ds_nodash}};
+DROP FOREIGN TABLE IF EXISTS topup_table_{{ds_nodash}};
+
+-- Vacuum or other upkeep here may be sensible?
```

Comment:
You _should_ get an autovacuum triggered anyway, and also won't need a vacuum to reclaim space after a table drop.

Reply:
:+1: